### ISSUE TITLE
Proposition de nom

### ISSUE THREAD

**Issue by claudiodsf** [IPGP/siqaco#1]
Quelques idée pour le nom de l'outil:

- QuaCo (Quality Control)
- QuaC (Quality Control)
- SQuaC (Seismic Quality Control)
- SQuC (prononcé comme l'italien "scusi" – Seismic Quality Control)
- SeQC (prononcé "sec" – Seismic Quality Control)
- YAQC (prononcé "yak" – Yet Another Quality Control)
- YASQC (prononcé "yask" – Yet Another Seismic Quality Control)
- ASQC (prononcé "ask" – Another Seismic Quality Control, ou Automatic Seismic Quality Control)
- BeQC (prononcé "beck" – Better Quality Control)

D'autres idées ? Des préférences ?

**Comment by claudiodsf** [comment]
- MASQC (prononcé "mask" – Mostly Automated Seismic Quality Control)

**Comment by claudiodsf** [comment]
- SEEDQC (SEED Quality Control)
- MSEEDQC (MiniSEED Quality Control)
- MSQC (MiniSEED Quality Control)

**Comment by geberoliv** [comment]
J'aurais bien vu un nom qui reste dans le domaine des plantes (puisque le logiciel va utiliser des objets et API qui ont des noms comme Seed, RabbitMQ, Celery, Flower...)

- SQUASH (Seed QUality control and Archive/Automatic SyncH.)
- SESAME (SEed Synchronizing Archive and Monitoring Environment)
- SPROUT (Seed data PROcessing UTility)

Le nom du logiciel a-t-il besoin d'être un acronyme? Parce que sinon on peut l'appeler comme on veut, tant qu'on explique pourquoi. Par exemple, j'aime bien le mot SPROUT, parce qu'il correspond à l'étape de croissance d'une plante entre la graine (SEED, les données) et l'arbre (TREE, l'arborescence finale). Mais pas sur que ce soit facile à assumer en français comme nom.

**Comment by claudiodsf** [comment]
SPROUT est génial, mais je suis perplexe par rapport à l’acronyme et, surtout, à la prononciation française. 

SESAME c’est très jolie aussi et en plus l’acronyme a du sens.

**Comment by geberoliv** [comment]
J'ai pensé à "SCIURUS" , comme le nom latin pour l'écureuil, car comme lui le projet collecte des graine (les fichiers seed) et les met en "cache" (archive) pour les utiliser plus tard ("QC, validation"), cf : [Hoarding sur Wikipédia](https://en.wikipedia.org/wiki/Hoarding_(animal_behavior)?oldid=798227454) . 

Ça pourrait aussi faire un joli logo (un écureuil...), et si le mot n'est pas un acronyme-je n'en ai pas trouvé, on peut lui ajouter un sous-titre comme par exemple "SCIURUS : a modular tool to collect and validate seismic data". 

Je pense que pour un projet open-source, on devrait avoir le droit d'utiliser ce mot ?

**Comment by geberoliv** [comment]
Un peu de manipulation parce que j'aime bien cette idée : 

![C'est mignon non?](https://upload.wikimedia.org/wikipedia/commons/f/fb/Ecureuil.jpg "C'est mignon non?")

**Comment by claudiodsf** [comment]
Why not using SQUIRREL : Seed QUality ... (to be completed) ?

Otherwhise, SCIURUS is nice but maybe a bit complicated to pronounce.

An alternative is the Greek form: skiouros (σκίουρος)

**Comment by geberoliv** [comment]
Yes, I tought so first, but it's already the name of an object oriented language [Wikipédia Squirrel](https://en.wikipedia.org/wiki/Squirrel_(programming_language)) and a SQL Client [Wikipédia SQL Client](https://en.wikipedia.org/wiki/SQuirreL_SQL_Client) ... so I dropped the idea

The Greek name is nice too, especially written in the Greek alphabet !
